modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/CliRequests.java

```diff
@@ -69,6 +69,11 @@ public interface ChangePeersRequest extends Message {
         String leaderId();
 
         Collection<String> newPeersList();
+
+        Collection<String> newLearnersList();
+
+        // term is intentionally Long and not long in order to perform nullable (not initialized) check.
+        Long term();
```

Comment:
Why isn't this field marked as `@Nullable`?

Reply:
Because it's mandatory.

Reply:
ok, I didn't understand the original intention, sorry